src/Sarif.FunctionalTests/ConverterTestData/StaticDriverVerifier/cancelspinlock_bug1.tt.sarif

```diff
@@ -52,13 +52,15 @@
                     },
                     {
                       "step": 1,
```

Comment:
I see that the steps are still 0-based (note that the first location in this flow has no "step" property, so it's 0). Is there another commit coming?